x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheUtils.java

```diff
@@ -33,14 +33,25 @@ public static int toIntBytes(long l) {
     }
 
     /**
-     * Rounds the length up so that it is aligned on the next page size (defined by SharedBytes.PAGE_SIZE). For example
+     * Round down the size to the nearest aligned size &lt;= size.
+     */
+    public static long roundDownToAlignedSize(long size, long alignment) {
+        return size / alignment * alignment;
+    }
+
+    /**
+     * Round up the size to the nearest aligned size &gt;= size
+     */
+    public static long roundUpToAlignedSize(long size, long alignment) {
+        return (((size - 1) / alignment) + 1) * alignment;
```

Comment:
Nit: Since this is now a standalone helper method. I wonder whether it is worthwhile to have some assertions such as `assert size > 0 && alignment > 0`.